pxr/usd/usdGeom/schema.usda

```diff
@@ -2292,6 +2292,36 @@ class Camera "Camera" (
     However, it follows that if even one property is authored in the correct
     scene units, then they all must be.
 
+    \\section UsdGeom_CameraExposure Camera Exposure Model
+
+    UsdGeomCamera models exposure by a camera in terms of exposure time, ISO,
+    f-number, and exposure compensation, mirroring the controls on a real camera.
```

Comment:
did we decide to normalize on "f-stop" rather than "f-number"? I think this is a lingering reference.

Reply:
We did, and this should be updated.

Reply:
I've pushed commits that fix this